ifupdown2/addons/address.py

```diff
@@ -1212,3 +1248,51 @@ def run(self, ifaceobj, operation, query_ifaceobj=None, ifaceobj_getfunc=None):
         else:
             op_handler(self, ifaceobj,
                        ifaceobj_getfunc=ifaceobj_getfunc)
+
+    def _settle_dad(self, ifaceobj, ips):
+        """ Settle dad for any given ips """
+        def ip_addr_list(what):
+            raw = json.loads(utils.exec_commandl([
+                'ip', '-j', '-o', '-6', 'address', 'list', 'dev',
+                ifaceobj.name, what
+            ]))
+            addr_infos = (x for t in raw for x in t.get('addr_info', []))
+            ip_list = ['%s/%s' % (x["local"], x["prefixlen"]) for x in addr_infos if x]
```

Comment:
Yes another nitpick:

Yup this does not change anything but it's priettier :)

```suggestion
            ip_list = ['{local}/{prefixlen}'.format(**x) for x in addr_infos if x]
```

Reply:
That it is